projectroles/forms.py

```diff
@@ -507,8 +507,14 @@ def __init__(self, project=None, current_user=None, *args, **kwargs):
             # Set owner
             if self.current_user.is_superuser and parent_project:
                 self.initial['owner'] = parent_project.get_owner().user
+                self.fields[
+                    'owner'
+                ].label_from_instance = lambda x: x.get_form_label()
             else:
                 self.initial['owner'] = self.current_user
+                self.fields[
```

Comment:
Note that this branch still contains this older version of the user dropdown fix implementation. We don't want to override the existing already merged solution with this one.

Reply:
I deleted all previous changes concerning owner name bug. However I do not understand, why do the github showed changes between branches. Dev is now merged with fix/owner-name )(